src/themes/teams/components/Icon/svg/types.d.ts

```diff
@@ -2,15 +2,16 @@ import { SvgIconSpec, ComponentSlotStyleFunction } from '../../../../types'
 import { ObjectOf } from '../../../../../../types/utils'
 import { IconProps } from '../../../../../components/Icon/Icon'
 
-type SvgIconSpecWithStyles = {
+type SvgIconSpecWithMetaData = {
```

Comment:
what if we will end up with these names:
- `SvgIconSpec` -> `SvgIconPathSpec`
- `SvgIconSpecWithStyles` -> `SvgIconSpec`

It seems that those will be more descriptive and, thus, easier to maintain.

Reply:
I reverted the changes in the public API as you suggested and I am adding classes for these icons